src/EntityManager.php

```diff
@@ -198,6 +198,27 @@ public function getList(){
 		return \$this->findAll();
 	}
 	
+	/**
+     * Finds only one entity. The criteria must contain all the elements needed to find a unique entity.
+     * Throw an exception if more than one entity was found.
+     *
+     * @param array \$criteria
+     *
+     * @return ".$entityName." the bean object
+     */
+    public function findTheOnlyOneBy(array \$criteria)
```

Comment:
Naming proposal:
what about naming the method: "findUniqueBy" rather than "findTheOnlyOneBy"?
It is a bit shorter.